ui/page/staking/purchase_modal.go

```diff
@@ -72,7 +78,7 @@ func newStakingModal(l *load.Load) *stakingModal {
 
 	tp.stakeBtn.SetEnabled(false)
 
-	tp.vspIsFetched = len((*l.WL.VspInfo).List) > 0
+	tp.vspIsFetched = len(l.WL.MultiWallet.VspList) > 0
```

Comment:
Retire this `vspIsFetched` property please and use `len(l.WL.MultiWallet.VspList) > 0` instead.